book.rb

```diff
@@ -0,0 +1,15 @@
+require './rental'
```

Comment:
- [ ] Please kindly note that your app did not function as it was in the previous review due to the require './rental' method that you newly added. **Note: You can remove it for the app to function as it was**.
 I used this snippet that you implemented to check your app as well. Thanks.
```
person = Person.new(22, 'mayito')
book = Book.new('comic', 'author')
rent = Rental.new('12-02-2023', person, book)
p rent
```